firmware/config/boards/hellen/alphax-2chan/compile_firmware.sh

```diff
@@ -3,4 +3,4 @@
 export USE_OPENBLT=yes
 
 cd ..
-bash ../common_make.sh hellen/alphax-2chan ARCH_STM32F4
+bash ../common_make.sh alphax-2chan ARCH_STM32F4 config/boards/hellen/alphax-2chan
```

Comment:
@andreika-git that's sad :( that's very error-prone. We need something better eventually